main.go

```diff
@@ -116,6 +116,8 @@ func main() {
 		clicommand.BootstrapCommand,
 	}
 
+	app.ErrWriter = os.Stderr
```

Comment:
app.Reader is also not set by NewApp, which we could potentially use. 
For completeness should we ensure all three are set appropriately?

Reply:
I think v1.22.10 does not have a Reader, so just setting the ErrWriter should be enough for now.